### ISSUE TITLE
Internal(?) type encoding displayed in inspection message

### ISSUE THREAD

**Issue by iakoder** [kalessil/phpinspectionsea#597]
- Run Array and string offset validity inspection
- Look at results
Some messages are fine, they end in... ([bool]), ([bool, array])
Some messages are less than helpful, ending in stuff like... ([?#E#M#C\Editor.Template_Expand, array])

I assume that's an internal encoding.   In this case the line in question is

```
        $_params = $this->Template_Expand($_params);

        $nparams['layout'] = isset($_params['layout'])
            ? $_params['layout']
            : ($_params['pages'])
                ? array('pages' => $_params['pages'])
                : array('fields' => $_params['fields']);

        if (!isset($nparams['layout']['entity'])) {   <--- here's where the inspection is triggered

```
Let me know if this is a PhpStorm issue.

PhpStorm 2017.3 EAP
Build #PS-173.3302.10, built on October 18, 2017
PhpStorm EAP User
Expiration date: November 17, 2017
JRE: 1.8.0_152-release-1024-b5 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.12.5

Php Inspections (EA Extended) 2.3.11

**Comment by kalessil** [comment]
Seems like some of types has not been resolved properly. Will fix in the next release.
